packages/d3fc-webgl/src/buffers/vertexConstantAttributeBuilder.js

```diff
@@ -31,32 +31,22 @@ export default () => {
                 target++;
             }
         }
-        for (
-            let index = componentsPerElement;
-            index < projectedData.length;
-            index += componentsPerElement
-        ) {
-            projectedData.copyWithin(
-                offset + index,
-                offset,
-                offset + componentsPerElement
-            );
-        }
         return projectedData;
     };
 
     const build = (gl, program, name, verticesPerElement, count) => {
         base(gl, program, name);
 
-        if (validSize >= count) {
```

Comment:
Why does this need to change from `count`?

Reply:
I don't know. It works either way so I'll change it to be consistent, though just in case, it's possible this appeared when Chris and I paired on it last Friday, do you remember if there was a particular reason for this? @chrisprice

Reply:
Actually makes most sense as verticesPerElement, changed.